src/eventEmitter/EventEmitter.ts

```diff
@@ -0,0 +1,23 @@
+import { EventName, EventListener, Token } from "../types";
+
+export class EventEmitter {
+  // eslint-disable-next-line @typescript-eslint/ban-types
```

Comment:
why is this eslint comment needed? Is `Function` correct for the type?

Reply:
should it be `(payload: unknown) => void`?

Reply:
i changed it to EventListener[] and payload to any